week_1/project/week_1.py

```diff
@@ -50,16 +50,22 @@ def get_s3_data(context):
     return output
 
 
-@op
-def process_data():
-    pass
-
+@op(
```

Comment:
Not critical, but you can also define the "In" for the op. I think it's not required since it can figure it out from the method's signature, but it seems to be useful? I'll post a question on slack :)

For example:
```
ins={"stocks": In(dagster_type=List[Stock])}
```

Reply:
Taken. Lets take the recommandation and. benefits for doing same